commit 6fa2e872ce528f957b0c3aa735aa46b24ed6601a
Author: Eric Anholt <eric@anholt.net>
Date:   Tue Dec 15 23:46:32 2015 +0000

    drm/vc4: Use "hpd-gpios" for HDMI GPIO, like what landed upstream.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
index cf5d5c9..da37483 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
@@ -68,7 +68,7 @@
 				      <0x7e808000 0x100>;
 				interrupts = <2 8>, <2 9>;
 				ddc = <&i2c2>;
-				hpd-gpio = <&gpio 46 GPIO_ACTIVE_HIGH>;
+				hpd-gpios = <&gpio 46 GPIO_ACTIVE_HIGH>;
 				clocks = <&cprman BCM2835_PLLH_PIX>,
 					 <&cprman BCM2835_CLOCK_HSM>;
 				clock-names = "pixel", "hdmi";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1117_6fa2e872ce528f957b0c3aa735aa46b24ed6601a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1117_6fa2e872ce528f957b0c3aa735aa46b24ed6601a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
